generated/java/gapic-google-longrunning-v1/build.gradle

```diff
@@ -16,13 +16,16 @@ repositories {
   mavenLocal()
 }
 
+compileJava.options.encoding = 'UTF-8'
+javadoc.options.encoding = 'UTF-8'
+
 dependencies {
-  compile 'com.google.api:gax:1.23.0'
-  testCompile 'com.google.api:gax:1.23.0:testlib'
-  compile 'com.google.api:gax-grpc:1.23.0'
-  testCompile 'com.google.api:gax-grpc:1.23.0:testlib'
+  compile 'com.google.api:gax:1.49.0'
```

Comment:
@vam-google Is this build.gradle change generated by some procedure or manually edited?